### DISCUSSION TITLE
Lack of instruction in the Chapter Hepatic impairment in learning center

### DISCUSSION THREAD

**Discussion by FishinW** [Open-Systems-Pharmacology/Forum#1500] [How-Tos]
I follow the instruction of the Chapter Hepatic impairment and I found some processes are not mentioned. The expression files of CYP1A2 and CYP3A4 were built after the individual files were built, but there is no content about adjust the CYP expression profile. All individual files were cloned based on the control files, which means that the expression of CYP1A2 and 3A4 was the control file.
![nfmwj1er](https://user-images.githubusercontent.com/128284693/230011598-34ca9509-1cc3-4501-8863-5186ce6ddcae.png)

**Comment by Yuri05** [comment]
@stephanschaller

**Comment by StephanSchaller** [comment]
@FishinW , are you referring to the exercise in the [esqLABS learning center](https://learn-esqlabs.talentlms.com/)?

Ecercise "[Hepatic Impairment](https://learn-esqlabs.talentlms.com/trainer/course/id:194)" describes here (Section "Expression Profile: Child-Pugh A, B and C") how to adapt the reference concentration:
![image](https://user-images.githubusercontent.com/25477643/230206438-1ea5e770-49ab-4779-aa3e-272dd76b1b58.png)

**Reply by FishinW** [reply]
Thank you for the reply.
The Section "Expression Profile: Child-Pugh A, B and C" did provide information about how to adapt the reference concentration.However, it was set behind the section "Individual: Child-Pugh C"
![image](https://user-images.githubusercontent.com/128284693/230386942-fe39fa18-7938-4251-a8a6-aebf1040a9cc.png)
As all individual files were cloned based on the control files, which means that the expression of CYP1A2 and 3A4 was the control file. There may should be another step of changing the CYP1A2/3A4 control expression file to relevant expression file (CYP3A4 Child-Pugh A/B/C) in the individual block.

**Reply by StephanSchaller** [reply]
Expression profiles and individuals are independent. So this is correct as described in the exercise

**Reply by HaiXxxxxx** [reply]
Dear @StephanSchaller ,
I want to know the relative concentrations of other enzymes such as CYP2C19 in different stages of liver damage. The materials in esqlabs are the concentrations of CYP1A2 and CYP3A4.
Thank you.

**Comment by StephanSchaller** [comment]
@Yuri05 , can you move this to the forum?

**Reply by claliposome** [reply]
I also find this issue during the training yesterday, I think for the individual create, the step is not correct, we need to build the enzyme at the disease state first and then incorperate it into the individual, then pool the population, while this course not mentioned or emphasized it when introduce the individual at the desease state. when we first go thourgh this course, it will cause an incorrect population and simulation thereafter, so that the final time profile is inconsistency with the course.
